src/components/student-record/student-record.tsx

```diff
@@ -0,0 +1,76 @@
+'use client'
+
+import { useState } from 'react'
+import StudentRecordDialog from './student-record-dialog'
+import StudentRecordFallback from './student-record-fallback'
+import { Student } from '@/lib/domain'
+
+const StudentRecord = ({
+  studentData,
+  pdfSrc,
+}: {
+  studentData: Student
+  pdfSrc: string
+}) => {
+  // Isso eh utilizado para atualizar o embed toda vez que atualizar o historico
+  const [pdfViewerKey, setPdfViewerKey] = useState(Math.random())
+
+  const dateRegex = /\d{4}-\d{2}-\d{2}/
+  let updatedAtFormatted = ''
+
+  if (studentData.academicRecordUrl) {
+    const updatedAtMatch = studentData.academicRecordUrl.match(dateRegex)
+
+    if (updatedAtMatch) {
+      const [year, month, day] = updatedAtMatch[0].split('-')
+      // Format the date as DD/MM/YYYY
+      updatedAtFormatted = `${day}/${month}/${year}`
+    }
+  }
```

Comment:
Uma sugestão aqui seria usar o timestamp no lugar da data no nome do arquivo do histórico. Assim nós poderíamos ter também a hora que foi atualizado o histórico. Além disso, poderia mudar também o formato para algo mais legível como: 28 de Jan. de 2024, 17:42. Apesar de que talvez ficasse muito grande.

De qualquer forma, somente uma sugestão.

Reply:
Acho que talvez o timestamp no nome do arquivo talvez deixasse ele muito longo e ruim para a leitura de quem baixasse o arquivo. Se for realmente necessário incluir a hora de atualização seria mais interessante ir pela abordagem de salvar este dado no modelo do banco. O que você acha?

Quanto ao formato mais legível eu cheguei a tentar transformar para data no JavaScript mas tive problemas quando ele fazia a transformação por conta do fuso horário e acabava voltando em um dia a data. Vou tentar resolver e testar se consigo.

Reply:
Spoiler: eu consegui kskksksks. Subi como commit. Ficou assim:
![image](https://github.com/Edge-Academy-UFAL/student-manager-ui/assets/108153768/606cd04f-e346-469c-881e-f96681d1694e)

Reply:
Ficou ótimo!